moped-editor/src/utils/dateAndTime.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Format date from the DB in YYYY-MM-DD format into human-readable locale date
+ * in MM/DD/YYYY format
+ * @param {String} dateString - Date string in date type ("2022-01-15") format
+ * @return {String} Date formatted as MM/DD/YYYY
+ */
+export const formatDateType = dateString => {
+  const dateObject = new Date(dateString);
+  return dateObject.toLocaleDateString("en-US", {
+    timeZone: "UTC",
+  });
+};
+
+/**
+ * Format timestampz string from the DB ("2022-05-17T22:37:26.072259+00:00")
+ * into human-readable locale date in MM/DD/YYYY format
+ * @param {String} timeStampZString - Timestampz string in ISO format
+ * @return {String} Date formatted as MM/DD/YYYY
+ */
+export const formatTimeStampZType = timeStampZString => {
```

Comment:
should this be `formatTimeStampTZType`? Seems consistent throughout at least.

![Screen Shot 2022-05-20 at 5 31 13 PM](https://user-images.githubusercontent.com/5697474/169620604-f37b9a91-1128-4380-a126-7d518d49bd52.png)

Reply:
Good call, yes, I will update this naming with TZ. I also added the run test script command to the PR description.